content/en/blog/ksynth-k8s-operator.md

```diff
@@ -0,0 +1,134 @@
+---
+date: 2022-01-17
+author: "Evan Hazlett"
+title: "Running Synthetic Checks at Scale with Kubernetes"
+slug: "ksynth-checks-k8s"
+summary: "Odyssey simplifies running synthetic checks across your organization with Kubernetes."
+---
+
+Ensuring your application is available, healthy, and performant is vital for any
+production service. Monitoring and observability critical component to notify when an application
+has an issue but what about the data leading up to that threshold? Synthetics give
+you continuous data points on the factors that matter most to your application whether
+it be a simple TCP ping, an HTTP status, or network trace. Does your application have
+increasing load times between application versions? Is there a spike in traffic at certain hours
+that trigger slower responses? Are there other processes that are running that are negatively
+affecting the service but not to the point where it falls over? As Kubernetes use increases,
+neighboring applications can cause significant issues if not configured properly.
+When synthetics are used in combination with your monitoring and observability stacks, they provide
+a powerful view into the overall health of the application and help identify issues
+before they cause an outage.
+
+## Synthetics
+
```

Comment:
I feel like these two paragraphs kinda say the same thing? Abstract and Introduction to a paper.